lib/composers/thenComposer.js

```diff
@@ -27,7 +28,9 @@ const makeThenComposer = (getSocketMessage, callback, beforeSend) => makeMethodC
 						return callback(error, data, socketMessage);
 					}))
 					.catch(err => {
```

Comment:
I think it was designed to handle only error from authorizeWs that can be only AUTH_NOT_ALLOWED, did you check maybe the problem somewhere else, some Fn throws error that is not expected?

Reply:
The point of this catch is displaying proper message if user not autorized, if some error happnes in call back function - it should be displayd properly.